manual/source/examples/python/simple_example_basic/index.rst

```diff
@@ -20,45 +20,50 @@ Since we are not using the NAPI, our
 support library must create and set the ``NX_class`` attribute on each group.
 
 .. note:: We want to create the desired structure of
-          ``/entry:NXentry/mr_scan:NXdata/``. 
-          
-	  #. First, our support library calls 
-             ``f = h5py.File()`` 
-             to create the file and root level NeXus structure.
-	  #. Then, it calls 
-             ``nxentry = f.create_group("entry")`` 
-             to create the ``NXentry`` group called
-             ``entry`` at the root level. 
-	  #. Then, it calls 
-             ``nxdata = nxentry.create_group("mr_scan")`` 
-             to create the ``NXentry`` group called
-             ``entry`` as a child of the ``NXentry`` group.
-
-Next, we create a dataset called ``title`` to hold a title string that can
-appear on the default plot.
-
-Next, we create datasets for ``mr`` and ``I00`` using our support library.
-The data type of each, as represented in ``numpy``, will be recognized by
-``h5py`` and automatically converted to the proper HDF5 type in the file.
-A Python dictionary of attributes is given, specifying the engineering units and other
-values needed by NeXus to provide a default plot of this data.  By setting ``signal="I00"``
-as an attribute on the group, NeXus recognizes ``I00`` as the default
-*y* axis for the plot.  The ``axes="mr"`` attribute on the :ref:`NXdata` 
+    ``/entry:NXentry/mr_scan:NXdata/``.
+
+    Examine the example code below for these key actions to create the structure:
+
+    =====================================   ================================================  ================================================
+    action                                  nexusformat function                              h5py function
+    =====================================   ================================================  ================================================
+    create file & root (``/``) structure    ``with nxopen(fileName, "w") as f:``              ``with h5py.File(fileName, "w") as f:``
+    create ``/entry`` group                 ``f["entry"] = NXentry()``                        ``nxentry = f.create_group("entry"); nxentry.attrs["NX_class"] = "NXentry"``
+    create ``/entry/mr_scan`` group         ``f["entry/mr_scan"] = NXdata(y, x)``             ``nxentry.create_group("mr_scan"); nxentry["mr_scan"].attrs["NX_class"] = "NXdata"``
+    store ``mr`` data                       ``x = NXfield(mr_arr, units="degrees", ...)``     ``nxdata.create_dataset("mr", data=mr_arr, units="degrees")``
```

Comment:
I don't recognise that `units` keyword

Reply:
I'll refactor that to use `ds.attrs["units"]=...` similar to how groups were done above.